index.html

```diff
@@ -857,111 +872,94 @@ <h2>
+        </li>
+        <li>
+          <a href="">
+          Threat Model for Decentralized Identities</a> [[threat-model-decentralized-identities]]
+        </li>
+        <li>
+          <a data-cite="vc-data-model#privacy-considerations">
+          VC Data Model Privacy Considerations</a>
+        </li>
+        <li>
+          ...
+        </li>
+      </ul>
+      <p>
+        Instead, this specification aims to describe where the Digital Credentials
+        API itself is responsible for meeting user expectations of Privacy, and
+        how developers and user agents implementing or integrating with the API
+        are to consider these responsibilities.
+      </p>
+      <p>
+        The privacy considerations for digital credentials are not static. They
+        will evolve over time as the ecosystem matures, and may be informed by
+        the behavior of other actors in the ecosystem, improvements in other
+        layers of the stack, new threats to user privacy, as well as changing
+        societal norms and regulations.
+      </p>
+      <p>
+        It is expected that the various groups involved in the design and implementation
+        of the Digital Credentials API actively monitor the evolving privacy
+        landscape and participate in the corresponding evolution of the API.
+      </p>
+      <section data-cite="vc-data-model#spectrum-of-privacy">
         <h3>
-          Data Retention
+          Spectrum of Privacy
         </h3>
-        <p class="issue" title="Work in progress">
-          Explain how verifiers might retain data and what the ecosystem does
-          to mitigate excessive data retention policies.
+        <p>
+          The Digital Credentials API serves a variety of use cases with different
+          grades of data disclosure and individual users with different preferences
+          depending on the context that they are in. Notably, the privacy properties
+          of a credential exchange mediated by this API could be mandated by
+          the legal and regulatory environment of an individual user.
         </p>
-      </section>
-      <section>
-        <h3>
-          Compliance with Privacy Regulations
-        </h3>
-        <p class="issue" title="Work in progress">
-          Explain to what extent the API complies with known privacy
-          regulations (e.g., consent) and what parts of those regulations are
-          not possible to enforce via the API (e.g., retention).
+        <p>
+          This means that some users may not want, or be allowed,
+          to use the most privacy-preserving means of exchanging credential information.
+          Nonetheless, user agents need to serve users with an experience
+          that is private by default and protect them from harm.
         </p>
-      </section>
-      <section>
-        <h3>
-          Selective and Unlinkable Disclosure
```

Comment:
Selective disclosure and unlinkable presentation are key privacy properties for any digital ID system. I would expect the privacy considerations section to detail the requirements for request protocols to enable selective disclosure to take place, and a way for linkable and unlinkable presentations to be distinguished in order to help the user identify tracking capabilities that they might not otherwise realize.

Reply:
Yeah, I agree that we should encourage user agents to take advantage of selective / unlinkable disclosures to frame their UI in a way that shows users the value of these properties and clearly communicates what will *not* be shared as part of an exchange that makes use of them.